app/admin/consultation_interactions.rb

```diff
@@ -0,0 +1,4 @@
+ActiveAdmin.register Consultation::Interaction do
```

Comment:
This file is generated by running the rake task as follows:
`bundle exec rake generate_active_admin_resources[consultation]`